Dockerfile

```diff
@@ -1,3 +1,20 @@
-FROM botpress/server:v12_5_0
```

Comment:
Mhh this is pretty outdated. Obviously that is another Dockerfile (for heroku in this case) which should be separately maintained. Where should these Dockerfiles actually be placed?

Reply:
Dockerfiles should be placed in the https://github.com/botpress/botpress/tree/master/build directory